freshmaker/handlers/botas/botas_shipped_advisory.py

```diff
@@ -82,13 +91,41 @@ def handle(self, event):
             self.log_info(msg)
             return []
 
+        if isinstance(event, ManualBundleRebuild):
+            bundles_to_rebuild = self._handle_manual_rebuild(db_event)
+        else:
+            bundles_to_rebuild = self._handle_auto_rebuild(db_event)
+
+        if not bundles_to_rebuild:
+            return []
+
+        builds = self._prepare_builds(db_event, bundles_to_rebuild)
+
+        # Reset context to db_event.
+        self.set_context(db_event)
+
+        self.start_to_build_images(builds)
+        msg = f"Advisory {db_event.search_key}: Rebuilding " \
```

Comment:
I don't see where `search_key` is set for `ManualBundleRebuild` and it's a required column in the database.

@qixiang what do you think about having `search_key` by a comma separated list of the values in `container_images`?

Reply:
Well, `search_key` for `ManualBundleRebuild` is inherited from `BaseEvent`. And there it's actually `msg_id`. And `msg_id` created in `FreshmakerManualRebuildParser.parse_post_data()` and it's generated by FM.